cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -43,4 +43,12 @@ function up() {
         $kubectl create -f /opt/cnao/network-addons-config-example.cr.yaml
         $kubectl wait networkaddonsconfig cluster --for condition=Available --timeout=200s
     fi
+
+    if [ "$KUBEVIRT_WITH_JUMBO_FRAMES" == "true" ];then
```

Comment:
Hey note that according to calico docs the patching new MTU would affect new workloads https://docs.projectcalico.org/networking/mtu#configure-mtu
We need to verify it also change all other pods MTU that created before the patch.
If this is the case you can add another change to calico manifest 
 https://github.com/kubevirt/kubevirtci/blob/master/cluster-provision/k8s/1.18/manifests/cni.diff